src/js/providers/html5.js

```diff
@@ -446,29 +450,15 @@ define([
                     _delayedSeek = -1;
                     _this.seek(startTime);
                 }
-
-                _play();
             }
 
             _position = _videotag.currentTime;
 
-            if (_isMobile && !hasPlayed) {
-                // results in html5.controller calling video.play()
-                _sendBufferFull();
```

Comment:
Since we only call `_sendBufferFull` after the `canPlay` event, we can get rid of some code in the `_sendBufferFull` function. We no longer need to check if we're on iOS and we may also be able to get rid of the `_canPlay` flag:
```
if (!_bufferFull && (!utils.isIOS() || _canPlay)) {
    ...
}
```